.gitignore

```
@@ -0,0 +1,3 @@
+.vscode
+*.csv
+*.png
```

Missing newline:
```suggestion
*.png

```
Also, you should probably add `Manifest.toml` and the `docs/build` directory